lib/virtio/virtqueue.c

```diff
@@ -63,11 +63,11 @@ int32_t virtqueue_create(uint16_t id,
 {
     struct virtqueue *vq    = VQ_NULL;
     volatile int32_t status = VQUEUE_SUCCESS;
-    uint32_t vq_size        = 0;
+    uint32_t vq_size;
```

Comment:
Why removing the vq_size initial value? I would rather changed it to = 0U;

Reply:
The value is initialized with first usage of it `vq_size = sizeof(struct virtqueue);`
That is why misra is complaining. So it depends how much you want implement misra checks.

Reply:
I guess when using 0U initial value, the Misra should be OK. What Misra rule this relates to?

Reply:
With `uint32_t vq_size =0U;` still "Variable 'vq_size' is assigned a value that is never used."
But sorry this one is not misra issue but cppcheck issue which i am also using. So you can ignore it if you want.

Reply:
OK, I would rather stay with current code in this case, our Misra check does not indicate any issue.

Reply:
Ok i will add U as you suggested. Misra is likely not complaining because value is not used...